tests/test_due_review_621.py

```
@@ -0,0 +1,810 @@
+    write_due_staged_commitment_todos(db, state)
+    db.conn.execute(
+        "UPDATE next_audience_todos SET created_turn=?",
+        (state.turn - 1,),
+    )
+    db.conn.commit()
+    apply_pending_due_reviews(db, state, commit=True)
+
+    dossier = db.get_decree_dossier(dossier_id)
+    assert dossier["execution_outcome"] in {"fulfilled", "degraded", "failed", "transformed"}
+    assert dossier["status"] == "closed"
+    # 连坐至多一次：整笔幂等门闩 cost_kind=liability 至多一行
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)